src/toggle/toggle.component.ts

```diff
@@ -105,8 +105,8 @@ export class Toggle extends Checkbox {
 	 * Text that is set on the left side of the toggle.
 	 */
 	@Input()
-	set offText(value: string) {
-		this._offText.next(value);
+	set offText(value) {
+		this._offText.next(value.OFF);
```

Comment:
why `value.OFF`? That would require passing `[offText]="{OFF: 'text`}"`, the value _should_ be a string.

Reply:
In any event this should be sorted by https://github.com/IBM/carbon-components-angular/pull/691